README.md

```diff
@@ -81,8 +81,8 @@ of [Go toolchain](https://golang.org/doc/install) (1.15 or 1.16 are supported).
 
 - __Linux__ is the default OS for which hivemind is developed and tested. We recommend Ubuntu 18.04+ (64-bit), but
   other 64-bit distros should work as well. Legacy 32-bit is not recommended.
-- __macOS 10.x__ can run hivemind using [Docker](https://docs.docker.com/desktop/mac/install/).
-  We recommend using [our Docker image](https://hub.docker.com/r/learningathome/hivemind).
+- __macOS__ is also supported, but some features may not work properly.
+  If you have issues, you can run hivemind using [our Docker image](https://hub.docker.com/r/learningathome/hivemind).
```

Comment:
Maybe it's best to keep the part about Docker in general, as some people would like to build their own image